e2e/actions.go

```diff
@@ -1,13 +1,52 @@
 package e2e
 
 import (
+	"encoding/json"
 	"log"
 	"math/rand"
 	"time"
 
 	"github.com/0xPolygon/pbft-consensus"
 )
 
+const (
+	DropNode       string = "DropNode"
+	RevertDropNode string = "RevertDropNode"
+	Partition      string = "Partition"
```

Comment:
Do we need this? As far as I remember, this is only required for node drop and we do not expect to do replay of other actions.